app/containers/ChangePasswordRequired.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { useDispatch } from 'react-redux';
+
+import { logout, setUser } from '../actions/login';
+import I18n from '../i18n';
+import { useSetting } from '../lib/hooks/useSetting';
+import { showErrorAlert } from '../lib/methods/helpers';
+import { Services } from '../lib/services';
+import { useTheme } from '../theme';
+import sharedStyles from '../views/Styles';
+import { useActionSheet } from './ActionSheet';
+import ActionSheetContentWithInputAndSubmit from './ActionSheet/ActionSheetContentWithInputAndSubmit';
+import Button from './Button';
+import { CustomIcon } from './CustomIcon';
+
+export const ChangePasswordRequired = () => {
+	const [loading, setLoading] = useState(false);
+	const { colors } = useTheme();
+	const dispatch = useDispatch();
+	const { showActionSheet, hideActionSheet } = useActionSheet();
+
+	const requiresPasswordConfirmation = useSetting('Accounts_RequirePasswordConfirmation');
+	const passwordPlaceholder = useSetting('Accounts_PasswordPlaceholder') as string;
+	const passwordConfirmationPlaceholder = useSetting('Accounts_ConfirmPasswordPlaceholder') as string;
+
+	const changePassword = async (password: string) => {
+		setLoading(true);
+		try {
+			await Services.setUserPassword(password);
+			dispatch(setUser({ requirePasswordChange: false }));
+			hideActionSheet();
+		} catch (error: any) {
+			showErrorAlert(error?.reason || error?.message, I18n.t('Oops'));
+		}
+		setLoading(false);
+	};
+
+	const showActionSheetPassword = () => {
+		const inputs = [{ placeholder: passwordPlaceholder || I18n.t('Password'), secureTextEntry: true, key: 'password' }];
+		if (requiresPasswordConfirmation) {
+			inputs.push({
+				placeholder: passwordConfirmationPlaceholder || I18n.t('Confirm_your_password'),
+				secureTextEntry: true,
+				key: 'confirm-password'
+			});
+		}
+		showActionSheet({
+			children: (
+				<ActionSheetContentWithInputAndSubmit
+					title={I18n.t('Please_Enter_your_new_password')}
+					testID='change-password-required-sheet'
+					inputs={inputs}
+					onSubmit={input => changePassword(input[0])}
+					isDisabled={input => (loading || input[0] === '' || requiresPasswordConfirmation ? input[0] !== input[1] : false)}
+				/>
+			)
+		});
+	};
+
+	return (
+		<View style={[styles.container, { paddingTop: 120, backgroundColor: colors.surfaceLight }]}>
+			<View style={styles.iconContainer}>
+				<CustomIcon name='info' size={36} color={colors.statusBackgroundWarning} />
```

Comment:
Wdyt of changing this color?

```suggestion
				<CustomIcon name='info' size={36} color={colors.statusFontWarning} />
```

### Before

<img width="505" alt="Screenshot 2024-06-05 at 1 36 59 PM" src="https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/804994/a3783efc-8a4c-4940-aa15-a6c7aa194c97">
<img width="505" alt="Screenshot 2024-06-05 at 1 36 47 PM" src="https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/804994/667678e8-35f9-4b46-a905-236bad3d5207">


### After

<img width="505" alt="Screenshot 2024-06-05 at 1 36 27 PM" src="https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/804994/4ad79c2a-ea4c-4dd8-8bd9-4a83746d4d1b">
<img width="505" alt="Screenshot 2024-06-05 at 1 36 40 PM" src="https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/804994/0f789512-743b-4ac9-8d8a-adb1f29b60ee">